config/default/translations/messages-en.properties

```diff
@@ -131,4 +131,428 @@ task.pnc.danger_sign_followup_mother.title = PNC danger sign follow up - mother
+report.delivery.babys_condition.baby_repeat.1.baby_details.vaccines_received = Vaccines received
+report.delivery.babys_condition.baby_repeat.1.baby_details.imm_counsel_note = Please counsel the family and refer the child to the nearest health center where the baby can be immunized.
+report.delivery.babys_condition.baby_repeat.1.baby_details.breatfeeding = Child exclusively breastfeeding
+report.delivery.babys_condition.baby_repeat.1.baby_details.breastfed_within_1_hour = Initiated on breastfeeding within on hour of delivery.
+report.delivery.babys_condition.baby_repeat.1.baby_details.baby_danger_sign_note = Danger signs
+report.delivery.babys_condition.baby_repeat.1.baby_details.infected_umbilical_cord = Infected umbilical cord
+report.delivery.babys_condition.baby_repeat.1.baby_details.convulsion = Convulsions
+report.delivery.babys_condition.baby_repeat.1.baby_details.difficulty_feeding = Difficulty feeding or drinking
+report.delivery.babys_condition.baby_repeat.1.baby_details.vomit = Vomits everything
+report.delivery.babys_condition.baby_repeat.1.baby_details.drowsy = Drowsy or unconscious
+report.delivery.babys_condition.baby_repeat.1.baby_details.stiff = Body stiffness
+report.delivery.babys_condition.baby_repeat.1.baby_details.yellow_skin = Yellow skin color
+report.delivery.babys_condition.baby_repeat.1.baby_details.fever = Fever
+report.delivery.babys_condition.baby_repeat.1.baby_details.blue_skin = Blue skin color
+report.delivery.babys_condition.baby_repeat.1.baby_details.r_baby_danger_sign_present = Danger Signs Present
+report.delivery.babys_condition.baby_repeat.1.baby_details.child_doc = Baby uuid
+report.delivery.babys_condition.r_baby_danger_sign_present_joined = Baby Danger signs
+report.delivery.babys_condition.r_baby_danger_sign_present_any = Any Baby Danger Signs Present
+report.delivery.pnc_visits = PNC Visits
+report.delivery.pnc_visits.who_note =  
+report.delivery.pnc_visits.pnc_visits_attended = PNC visits done so far
+report.delivery.pnc_visits.pnc_visits_additional = Additional PNC visits 
+
+report.pregnancy_facility_visit_reminder.facility_visit_reminder = Reminder of Upcoming Health Facility Visit
+report.pregnancy_facility_visit_reminder.facility_visit_reminder.remind_note=
+report.pregnancy_facility_visit_reminder.facility_visit_reminder.remind_method=Client Reminder Method
+report.pregnancy_facility_visit_reminder.facility_visit_reminder.custom_translations= Custom translations
+report.pregnancy_facility_visit_reminder.facility_visit_reminder.custom_translations.custom_woman_label_translator= Woman Label translation
+report.pregnancy_facility_visit_reminder.facility_visit_reminder.custom_translations.custom_woman_label= Woman label
+report.pregnancy_facility_visit_reminder.facility_visit_reminder.custom_translations.custom_woman_start_label_translator= Woman start label translator
+report.pregnancy_facility_visit_reminder.facility_visit_reminder.custom_translations.custom_woman_start_label= Woman start label
```

Comment:
These fields should be hidden from the form and not shown in the report view, so they shouldn't have labels here.

![image](https://user-images.githubusercontent.com/789512/68145642-41c6a400-ff04-11e9-82f1-ea1bd93b66ce.png)